examples/upload-media-from-local/upload.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+import os
+import sys
+
+sys.path.insert(0, os.getcwd())
+from examples.utils.request import handle_request, handle_authorization
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("Host", metavar="host", type=str, help="Host Url")
```

Comment:
done

Reply:
My original suggestion was bad and now it even has a typo :)  may be it could be `help="upload on behalf of a user",`
Now it would look like:
```
❯ bin/run examples/upload-media-from-local/upload.py -h
usage: upload.py [-h] [--user-id USER_ID] host files [files ...]
...
optional arguments:
  -h, --help         show this help message and exit
  --user-id USER_ID  upload on behalf of a user
```

Reply:
fixed

Reply:
Sorry, but I still can't see the help comment suggested above:
```
    parser.add_argument("--user-id", type=int, required=False
                       help="Optional parameter to upload on behalf a user")
```

Reply:
done